nl_server/requirements.txt

```diff
@@ -8,9 +8,6 @@ gunicorn==20.1.0
 markupsafe==2.0.1
 pandas==1.3.5
 sentence_transformers==2.2.2
-spacy==3.4.4
+spacy==3.5.0
 torch==1.12.1 # torch 1.13 or higher brings in nvidia lib, which  is 1.5GB.
-# Downloading the named-entity recognition (NER) library spacy and the large EN model
-# using the guidelines here: https://spacy.io/usage/models#production
```

Comment:
need to add this to nl_server/requirements.txt?

Reply:
Since the install for these is now being done in run_nl_server.sh there is no need to have them here, right? Also this is not a library but a data file packed like a library. So it's Ok to not call it a dependence?

Reply:
run_nl_server.sh is only used locally, for actual deployment, it will depend on the requirements.txt

Reply:
Note the python cloudbuild failure is due to this.

Reply:
hopefully it should be fixed now